boskos/deployment.yaml

```diff
@@ -11,7 +11,11 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: test-pods
+---
 apiVersion: v1
```

Comment:
I don't think the persistent volume is required anymore. @krzyzacy

Reply:
hummm I thought we still fall back to in-memory? (I haven't tried to get rid of it, your call)